.github/workflows/pr_lint.yml

```diff
@@ -0,0 +1,29 @@
+# Linting PR title in order to ensure semantic versioning
+# Adapted from: https://github.com/amannn/action-semantic-pull-request
+# Semantic versioning follows: https://www.conventionalcommits.org/
+
+name: PR title linter
+
+# Currently I have this set to only PRs made into this repo
+# As we accept community PRs through forks -> need to change this for
```

Comment:
sounds good, just for my own understanding @ishandhanani - this means that while we are able to accept PRs from forks without them having to follow our semantic guidelines, they will not be reflected in the `changelog` accordingly?